Assets/MixedRealityToolkit-SDK/Profiles/SpatialAwareness/DefaultWindowsMixedRealitySpatialMeshObserverProfile.asset

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 90a4bdbfdea331745b57570c107eeaae, type: 3}
+  m_Name: DefaultWindowsMixedRealitySpatialMeshObserverProfile
+  m_EditorClassIdentifier: 
+  isCustomProfile: 1
+  startupBehavior: 0
+  observationExtents: {x: 3, y: 3, z: 3}
+  isStationaryObserver: 1
```

Comment:
To [match the original value](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3148/files#diff-5b6ab8fcf25beaea64478b8dd17d1768L17) and the expectations of the demo scene, which describes the ability to move around and watch the mesh update around you.

```suggestion
  isStationaryObserver: 0
```